### DISCUSSION TITLE
Settings pages

### DISCUSSION THREAD

**Discussion by wwdes** [wpengine/atlas-content-modeler#391] [Ideas]
Have you considered adding functionality for creating settings pages?

**Comment by mindctrl** [comment]
Hi @wwdes, thanks for the question. Yes, that's a feature request we've had before and are considering. Would you mind sharing some of the types of settings pages you'd like to create?

**Reply by wwdes** [reply]
I was thinking of creating settings pages with their data exposed via rest api. This is so that editors can then make changes to content that does not use a repeatable model. For example, it can be a store notice or a paragraph of text on a landing page.

**Reply by mindctrl** [reply]
Got it. Thank you for the example. We'll update you here when we have more to share.

**Comment by Bowriverstudio** [comment]
I would also like a settings page - I've got two use cases.

1) Global settings for the site -  Ie: Email address, Phone Number, Address ...

Currently, I accompolish this with ACF - and call the page Theme Settings. 

2) Settings for a CPT 

In my "Archive" Page for a CPT, I have a paragraph of text I'd like to display before the list of the CPTs. 

Thanks for the consideration.
